app/code/community/Businessfactory/Roihuntereasy/controllers/CronController.php

```diff
@@ -66,6 +60,52 @@ public function indexAction()
     }
 
     /**
+     * http://store.com/roihuntereasy/cron/index
```

Comment:
Please edit the comments section while copy pasting. URL of this endpoint is not `cron/index`.
Also I would recommend to create a new controller for products preview instead of misusing `CronController`.